checkpoint/generator.py

```
@@ -6,95 +6,240 @@
+    prompt = get_agent_specific_prompt(
+        query=query,
+        context_str=context_str,
+        detected_intent=detected_intent,
+        agent_type=agent_type,
+        limit=limit,
+        has_context=bool(context_to_use)
+    )
 
     try:
+        # Log the enhanced request details for debugging
+        print(f"Generating checkpoints with enhanced context:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)